### PR TITLE
Update erotic.json

### PR DESCRIPTION
**Pull request by Asteliks**
added Erotic Store Venus. Before merge I need help creating the ID. Also I would love to become a maintainer if thats possible.

### CODE REVIEW

File: data/brands/shop/erotic.json [hunk 1]
```diff
@@ -95,6 +95,18 @@
...
+        {
+      "displayName": "Erotic Store Venus",
+      "locationSet": {"include": ["616"]},
+      "tags": {
+        "brand": "Erotic Store Venus",
+        "brand:wikidata": "Q122900757",
+        "name": "Erotic Store Venus",
+        "shop": "erotic"
+        "min_age": "18"
+        "website": "https://www.venus.net.pl/"
```
Comment (arch0345) [inline data/brands/shop/erotic.json:108]: Please note that we don't support websites in NSI (see [this wiki page](https://github.com/osmlab/name-suggestion-index/wiki/Contributing#tags-required))
```suggestion
```

**Review by arch0345 — CHANGES_REQUESTED** [review]

**Comment by arch0345** [comment]
Thanks for your interest in becoming a maintainer! However, I'm a bit reluctant to invite first-time contributors to this project. I might consider your request once you actively contribute.

In regards to creating ids, we have [a script that automatically generates them](https://github.com/osmlab/name-suggestion-index/wiki/Contributing#id-generated).

File: data/brands/shop/erotic.json (hunk 1 above)
Comment (UKChris-osm) [reply data/brands/shop/erotic.json:108]: Each tag also needs a comma at the end (except the last one) otherwise it causes a build error.

**Comment by Asteliks** [comment]
Thanks for the feedback concerning my PR. Of course I understand you're reluctant. It may help that I'm part of the official Polish chapter of OSM and a Collaborator at https://github.com/osmlab/editor-layer-index
My changes would be concentrated within the borders of Poland

**Review by arch0345 — APPROVED** [review]
Thanks for your first contribution to NSI!

**Pull request merged**